### PR TITLE
Use auto imported aria-common scripts, fix mapping table duplication and ids!

### PR DESCRIPTION
**Pull request by spectranaut**
***

### :boom: Error: 504 Gateway Timeout :boom: ###

[PR Preview](https://github.com/tobie/pr-preview#pr-preview) failed to build. _(Last tried on May 24, 2022, 5:49 PM UTC)_.

<details>
<summary>More</summary>

PR Preview relies on a number of web services to run. There seems to be an issue with the following one:

:rotating_light: [Spec Generator](https://www.w3.org/2015/labs/) - Spec Generator is the web service used to build specs that rely on ReSpec.

:link: [Related URL](https://labs.w3.org/spec-generator/?type=respec&url=https%3A%2F%2Fraw.githubusercontent.com%2Fw3c%2Fhtml-aam%2Fb88877f603becdd6b2605f75b018f89315510d5c%2Findex.html%3FisPreview%3Dtrue)

```
<html><body><h1>504 Gateway Time-out</h1>
The server didn't respond in time.
</body></html>

```

_If you don't have enough information above to solve the error by yourself (or to understand to which web service the error is related to, if any), please [file an issue](https://github.com/tobie/pr-preview/issues/new?title=Error%20not%20surfaced%20properly&body=See%20w3c/html-aam%23406.)._
</details>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -93,7 +79,6 @@
         "REC": "https://www.w3.org/TR/accname/"
       },
       xref: ["HTML", "core-aam", "accname", "wai-aria"],
-      localBiblio: localBiblio,
```
Comment (spectranaut) [inline index.html:96]: localBibilo.js just adds this attribute directly to the respecConfig object, now.

**Comment by spectranaut** [comment]
There is a failure on including the "github-actions[bot]" in the contributions from the HTML accessibility task force list... but I couldn't find the code for this, does someone know where it is?

**Comment by jnurthen** [comment]
> There is a failure on including the "github-actions[bot]" in the contributions from the HTML accessibility task force list... but I couldn't find the code for this, does someone know where it is?

This is in respec - https://github.com/w3c/respec/blob/471d6f5b20aab17e0b9db969418ef53f4b568d4c/src/core/contrib.js

**Comment by jnurthen** [comment]
filed https://github.com/w3c/respec/issues/4169 - please add any details, the bug is pretty bare bones but I have to run.

**Comment by jnurthen** [comment]
I have a hacky fix if you like in utility.js.
Add the following in utility.js
```
function fixContributors() {
	document.querySelectorAll("#gh-contributors li a").forEach(function(node){
		if (node.textContent.indexOf("[bot]") >  0) node.parentElement.parentElement.removeChild(node.parentElement);
	})
}
```
Then add 
```
      postProcess: [ fixContributors ],
```
into respecConfig in index.html

**Review by jnurthen — APPROVED** [review]

**Pull request merged**